### PR TITLE
chore: NNS proposals to upgrade ckERC20 ledger suite orchestrator and ckETH minter

### PR DESCRIPTION
**Pull request by gregorydemay**
([XC-176](https://dfinity.atlassian.net/browse/XC-176) and [XC-178](https://dfinity.atlassian.net/browse/XC-178)): NNS proposals to upgrade the ckERC20 ledger suite orchestrator and the ckETH minter to the latest version to fix a security issue in previous versions of `ic-cdk`.

[XC-176]: https://dfinity.atlassian.net/browse/XC-176?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[XC-178]: https://dfinity.atlassian.net/browse/XC-178?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_08_23_ckxaut.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Proposal to upgrade the ledger suite orchestrator canister to add ckXAUT
```
Comment (gregorydemay) [inline rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_08_23_ckxaut.md:1]: this file was simply rename in this [commit](https://github.com/dfinity/ic/pull/1075/commits/975a00a588a8e2aad3adfcd578c1fe916ca11106).

File: rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_08_23.md [hunk 2]
```diff
@@ -1,41 +1,43 @@
-# Proposal to upgrade the ledger suite orchestrator canister to add ckXAUT
+# Proposal to upgrade the ledger suite orchestrator canister
```
Comment (gregorydemay) [inline rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_08_23.md:1]: This is actually a new file, ignore the diff from the left. Git is confused with the rename from this [commit](https://github.com/dfinity/ic/pull/1075/commits/975a00a588a8e2aad3adfcd578c1fe916ca11106).

File: rs/ethereum/cketh/mainnet/minter_upgrade_2024_08_23.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# Proposal to upgrade the ckETH minter canister
+
+Git hash: `667a6bd3bc08c58535b8b63bfebc01dba89c0704`
+
+New compressed Wasm hash: `d6c0cf6adb923b70dd869b5b75b41e6b8db3ff2e0bb50a5e572353a94fade334`
```
Comment (ninegua) [inline rs/ethereum/cketh/mainnet/minter_upgrade_2024_08_23.md:5]: verified

File: rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_08_23.md [hunk 4]
```diff
@@ -1,41 +1,43 @@
...
 
-Git hash: `3d0b3f10417fc6708e8b5d844a0bac5e86f3e17d`
+Git hash: `667a6bd3bc08c58535b8b63bfebc01dba89c0704`
 
-New compressed Wasm hash: `ccda8966249bc7d89f14968a242d6fb2c33d2363336303ec4f12977cab74c6f7`
+New compressed Wasm hash: `fedd6d09b477eb6f79a682fc1868c97de04fe4469817374a8ddf8964f47485b2`
```
Comment (ninegua) [inline rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_08_23.md:5]: verified

**Review by ninegua — APPROVED** [review]

File: rs/ethereum/cketh/mainnet/minter_upgrade_2024_08_23.md (hunk 3 above)
Comment (THLO) [reply rs/ethereum/cketh/mainnet/minter_upgrade_2024_08_23.md:5]: Confirmed!

File: rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_08_23.md (hunk 4 above)
Comment (THLO) [reply rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_08_23.md:5]: Confirmed!

**Review by THLO — APPROVED** [review]

**Pull request merged**